api/src/utils/pagination/pagination-query.dto.ts

```diff
@@ -16,7 +16,7 @@ export type QuerySortDto<T> = [
 ];
 
 export type PageQueryDto<T> = {
-  skip: number | undefined;
-  limit: number | undefined;
-  sort: QuerySortDto<T>;
+  skip: number;
+  limit: number;
+  sort?: QuerySortDto<T>;
```

Comment:
Curious to know why we need to update this type.

Reply:
findAllQuery and findAll methods both have sort argument optional that's why